### PR TITLE
Scale testing

### PR DESCRIPTION
**Pull request by chrisstafford**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202062295771275

### CODE REVIEW

**Comment by chrisstafford** [comment]
kube-burner configs live in https://github.com/chrisstafford/docs/tree/scale-testing-config. Alerting configs will be in a third PR

**Comment by chrisstafford** [comment]
Tests have been run and verified against the production cluster, I'd like to get this merged so we can merge in the configs.

File: cluster_scaling_and_load_testing.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Cluster Scaling and load testing
+
+## Cluster Scaling
+
+We will initially be starting out with 3 Control nodes and 5 worker nodes. From here will will scale up at intervals of 10 worker nodes per until we reach the maximum number of available nodes.
```
Comment (joachimweyl) [inline cluster_scaling_and_load_testing.md]: `will will` -> we will
`per until` per what?

File: cluster_scaling_and_load_testing.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Interval Testing
+
+After each interval reports that the nodes are ready we will use [kube-burner][1]. The goal here is two-fold. First we want to make sure that we can schedule pods on each node, and second that we can max out node capacity without needing to add infra nodes to support the three controller nodes.
```
Comment (naved001) [inline cluster_scaling_and_load_testing.md:9]: > second that we can max out node capacity without needing to add infra nodes to support the three controller nodes.

Sorry, I don't understand what this means. Are we talking about adding nodes to the nerc-ocp-infra cluster?

File: cluster_scaling_and_load_testing.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+*0.6 Projects per worker node*
+
+*9 configMaps per worker node*
+
+*10 clusterIPServices per worker node*
+
+*11 secrets per worker node*
+
+*1.5 deploymenys and replicaSets per worker node*
```
Comment (naved001) [inline cluster_scaling_and_load_testing.md]: typo: deploymen**t**s

File: cluster_scaling_and_load_testing.md (hunk 2 above)
Comment (chrisstafford) [reply cluster_scaling_and_load_testing.md:9]: You can unload a lot of the admin tasks done by the controller nodes by adding infra nodes. These reduce the stress on the controllers, but can't be used to schedule regular workloads like worker nodes. I don't foresee us having any issues in the short term, but it's something that should be checked as the cluster and demand grows.

**Comment by chrisstafford** [comment]
Need to roll back last push, it included an unintended commit

File: cluster_scaling_and_load_testing.md (hunk 1 above)
Comment (chrisstafford) [reply cluster_scaling_and_load_testing.md]: Fixed, thank you

File: cluster_scaling_and_load_testing.md (hunk 2 above)
Comment (naved001) [reply cluster_scaling_and_load_testing.md:9]: ah, thanks for clarifying! It didn't occur to me that these are different class of nodes besides the worker and controllers.

**Review by naved001 — APPROVED** [review]

**Pull request merged**